requirements.txt

```diff
@@ -1,11 +1,13 @@
-pip~=22.0
+pip==23.0
```

Comment:
Does 23.0 work? Or was 23.1+ the problem?

Reply:
Yes, as far as I know problems begin at 23.1.